recipes/h3/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class H3Conan(ConanFile):
+    name = "h3"
+    description = "Hexagonal hierarchical geospatial indexing system."
+    license = "Apache-2.0"
+    topics = ("conan", "h3", "hierarchical", "geospatial", "indexing")
+    homepage = "https://github.com/uber/h3"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "h3_prefix": "ANY"
```

Comment:
do you really need it?

Reply:
Well, it's an option provided by the library. The default value disables prefix. I assume that consumers may expect this option even if it's an hack (but not a conan hack).